contracts/truefi/TrueLender.sol

```diff
@@ -74,6 +74,11 @@ contract TrueLender is ITrueLender, Ownable {
     // minimum prediction market voting period
     uint256 public votingPeriod;
 
+    // ===== End of pool parameters =====
```

Comment:
I don't get this comment, isn;t maxLoans also a pool parameter?

Reply:
Very good point. I probably tried to overdo it. Will fix this